hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -216,6 +218,29 @@ def collect_alembic_caches(self, parent_item, project_root):
                 cache_path
             )
 
+    def _collect_session_geometry(self, parent_item):
+        """
+        Creates items for alembic caches
```

Comment:
Suggest changing this to not mention alembics since publish plugin could export any type of file.

Reply:
Corrected